core/templates/components/question-directives/question-player/question-player-concept-card-modal.controller.spec.ts

```diff
@@ -129,7 +131,7 @@ describe('Question Player Concept Card Modal Controller', function() {
     expect($scope.modalHeader).toBe(undefined);
   });
 
-  it('should retry test', function() {
+  it('should retry test by changing window location', function() {
```

Comment:
Formulate as "it should [do X] [under condition Y]"

E.g. "it should refresh the page when the user retries a test" (or whatever the correct thing should be -- I'm just making this up).

Reply:
Done.